stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/MessagesScreen.kt

```diff
@@ -224,6 +225,18 @@ public fun MessagesScreen(
                 onDismiss = { listViewModel.dismissMessageAction(deleteAction) }
             )
         }
+
+        val flagAction = messageActions.firstOrNull { it is Flag }
+
+        if (flagAction != null) {
+            SimpleDialog(
```

Comment:
We should look into styling this dialog to match our theme if we can, for the sample. The accent color is purple at the moment and it looks out of place.